python/ray/experimental/state/common.py

```diff
@@ -516,6 +548,38 @@ class ClusterEventState(StateSchema):
     custom_fields: Optional[dict] = state_column(filterable=False, detail=True)
 
 
+class Humanify:
+    """A class containing default methods to convert units into a human readable string."""
+
+    def timestamp(x: float):
```

Comment:
Sure! Added a few helpers in test_sanity_check to parse output_string for a specific key and verify the value is of correct format. As a future TODO, it would be nice to figure out why json.loads(output_from_runner) does not work, and I wouldn't have to use a hacky regular expression solution.

Reply:
UPDATE: I removed the complex regex in favor of simpler testing methods from feedback.